package.json

```diff
@@ -78,13 +78,13 @@
     "@bbc/moment-timezone-include": "1.1.4",
     "@bbc/psammead-amp-geo": "1.2.2",
     "@bbc/psammead-assets": "3.1.5",
-    "@bbc/psammead-brand": "7.0.30",
+    "@bbc/psammead-brand": "7.1.1",
     "@bbc/psammead-bulleted-list": "3.0.12",
     "@bbc/psammead-bulletin": "5.0.30",
     "@bbc/psammead-byline": "3.0.13",
     "@bbc/psammead-calendars": "2.0.23",
     "@bbc/psammead-caption": "4.1.8",
-    "@bbc/psammead-consent-banner": "5.3.2",
+    "@bbc/psammead-consent-banner": "5.4.1",
```

Comment:
And with this component, we intend to rename the `focusRef` prop to `headingRef`.

Reply:
Please could you explain this decision? Is there a heading?

Reply:
> Please could you explain this decision? Is there a heading?

Yeah, the Consent Banner title is an `h2`. The variable `headingRef` refers to that element.

Reply:
Thanks